man/systemd.service.xml

```diff
@@ -898,6 +898,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestartQuick=</varname></term>
```

Comment:
This is the name that was suggested in PR https://github.com/systemd/systemd/pull/26751.

@poettering @DaanDeMeyer Are you ok with `NotifyFailureOnRestart=` instead of `RestartQuick=`?